packages/ember-glimmer/lib/renderer.js

```diff
@@ -159,31 +106,10 @@ class Renderer {
       rootOutletState: ref,
       isTopLevel: true
     });
-
-    let result, shouldReflush, afterRender, didChangeAfterRender;
-
-    let callback = () => {
-      result = view.template.asEntryPoint().render(self, env, { appendTo: target, dynamicScope });
-    };
-
-    let rerender = () => result.rerender({ alwaysRevalidate: shouldReflush });
-
-    do {
-      env.begin();
-      shouldReflush = runInTransaction(callback);
-      afterRender = CURRENT_TAG.value();
-      env.commit();
-      didChangeAfterRender = !CURRENT_TAG.validate(afterRender);
-      callback = rerender;
-    } while (shouldReflush || didChangeAfterRender);
-
-    this._scheduler.registerView(view);
-
-    return result;
+    this._renderRoot(view, view.template, self, target, dynamicScope);
   }
 
   appendTo(view, target) {
```

Comment:
Maybe we should rename target to parentElement since we have `targetObject` below (I was confused way further down and had to scroll back up)